code/datums/components/temporary_atom.dm

```diff
@@ -0,0 +1,13 @@
+/// Deletes the atom with a little fading out animation after a specified time
+/datum/component/temporary_atom
```

Comment:
This seems like it could be an element instead?

Reply:
I think initially I made a component because I thought I needed to hold references to the timers to delete them but `TIMER_DELETE_ME` probably handles that

Reply:
I'll be honest I'm no longer certain this is a component either given that all it does is start two timers but it's, _like_ one at least I guess.
Should I just make this into a global proc or is it good how it is?